cmd/cosign/cli/verify/verify_blob_test.go

```diff
@@ -239,12 +234,12 @@ func TestVerifyBlob(t *testing.T) {
 	otherSignature := makeSignature(otherBytes)
 
 	tts := []struct {
-		name        string
-		blob        []byte
-		signature   string
-		sigVerifier signature.Verifier
-		cert        *x509.Certificate
-		bundlePath  string
+		name       string
+		blob       []byte
+		signature  string
+		key        []byte
+		cert       *x509.Certificate
+		bundlePath string
 		// If online lookups to Rekor are enabled
 		experimental bool
```

Comment:
Most of these tests set the experimental flag, but we've begun to remove it - should we remove it from the tests too? (Separate refactor prolly)

Reply:
Yeah, I'll add in a separate PR: it actually does nothing now :)